rs/messaging/tests/queue_tests.rs

```diff
@@ -1,1107 +0,0 @@
-    into_env.execute_block_with_xnet_payload(xnet_payload);
-    Ok(())
-}
-
-/// Inducts just the stream header on `from_env` into `into_env`.
-fn induct_stream_header(
-    from_env: &StateMachine,
-    into_env: &StateMachine,
-) -> Result<(), EncodeStreamError> {
-    induct_from_head_of_stream(from_env, into_env, Some(0))
-}
-
-/// Calls the 'start' method on a canister in the state machine (assumed to be an XNet canister).
-fn call_start_on_xnet_canister(
-    env: &StateMachine,
-    canister_id: CanisterId,
-    payload: Vec<u8>,
-) -> Result<(), UserError> {
-    let wasm = env.execute_ingress(canister_id, "start", payload)?;
-    assert_eq!(
-        "started".to_string(),
-        Decode!(&wasm.bytes(), String).unwrap()
-    );
-    Ok(())
-}
-
-/// Calls the 'stop' method on a canister in the state machine (assumed to be an XNet canister).
-fn call_stop_on_xnet_canister(
-    env: &StateMachine,
-    canister_id: CanisterId,
-) -> Result<(), UserError> {
-    let wasm = env.execute_ingress(canister_id, "stop", Encode!().unwrap())?;
-    assert_eq!(
-        "stopped".to_string(),
-        Decode!(&wasm.bytes(), String).unwrap()
-    );
-    Ok(())
-}
-
-/// Triggers a timeout by first advancing the time by 1 day (which should far exceed any
-/// realistic setting for request lifetimes) and then ticking.
-fn execute_round_with_timeout(env: &StateMachine) {
-    env.advance_time(std::time::Duration::from_secs(24 * 3600));
-    env.tick();
-}
-
-/// Wrapper for a generic function to be executed at most a maximum number of times.
-/// The generic function must provide a condition to decide whether execution should be
-/// aborted or not. The condition is wrapped in `Result` so that the `?` operator can be used
-/// in the closure `f`.
-/// This is useful as a safety wrapper for code where we tick on a state machine
-/// until an exit condition is met. Since this usually depends on a canister that could stall
-/// or otherwise take an unreasonably long amount of time to reach the exit condition, this
-/// function makes sure a panic is triggered rather than hanging forever.
-/// Returns the number of iterations until the exit condition was met, or an error message
-/// indicating that the counter exceeded `max_iterations`.
-fn do_until_or_panic<F>(max_iterations: u64, mut f: F) -> Result<u64, String>
-where
-    F: FnMut(u64) -> Result<bool, String>,
-{
-    for counter in 1..=max_iterations {
-        if f(counter)? {
-            return Ok(counter);
-        }
-    }
-    Err(format!(
-        "Exit condition not met after {max_iterations} iterations."
-    ))
-}
-
-/// Test timing out requests in output queues, by completely filling up the XNet stream with
-/// messages from a local canister, such that messages start piling up in its output queue.
-/// Then advance the time and execute a round to trigger a timeout; then check the output queue
-/// has been emptied out.
-#[test]
-fn test_timeout_removes_requests_from_output_queues() {
```

Comment:
As far as I can tell this test and the test below are the only ones that are no longer explicitly covered by the new tests. I suppose one can argue that this is fine because this functionality is better tested as part of unit/integration tests anyways.